### DISCUSSION TITLE
Job Application Resources

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#539] [Job Resources]
This is a common question at Virtual Coffee and also the topic of our next monthly challenge. Remember, you don't have to be actively looking for a job to update your resume or cover letter or to work on an elevator pitch or interviewing skills. If you have a resource you'd recommend, drop it below.

**Comment by BekahHW** [comment]
- [ @abbeyperini's L&L on promoting yourself to potential employers](https://www.youtube.com/watch?v=NVaZu8--4p0)
- [@abbeyperini and Jennifer K's L&L Optimize Your Resume For Both A Hiring Manager And An ATS ](https://www.youtube.com/watch?v=XwPVZNvP_IM)
- [Breaking down coding problems in interviews – Courtney Landau – Lightning Talks 2021](https://www.youtube.com/watch?v=ghFNiCMy67M)
- [Resume Roundtable](https://www.youtube.com/watch?v=p4Ux7c7PQ0M)

**Comment by redapy** [comment]
Free resume [builder](https://app.flowcv.io/) that I found incredible!
